docs/security/how-to-authenticate.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "How to authenticate"
+slug: /security/how-to-authenticate
+keyword: security authentication oauth kerberos
+license: "This software is licensed under the Apache License version 2."
+---
+
+## Authentication
+
+Apache Gravitino supports three kinds of authentication mechanisms: simple,OAuth and Kerberos.
+
+### Simple mode
+
+Simple mode is the default authentication option of the server.
+
+For the client side, if it doesn't set the authentication explicitly, it will use anonymous to access the server.
+
+If the client sets the simple mode,  it will use the environment variable `GRAVITINO_USER` as the user.
```

Comment:
`GRAVITINO_USER` in the client-server....

Reply:
Done.

Reply:
Should we support setting a custom user name?

Reply:
Sure, Hadoop has the similar behaviour.